src/main/java/lotto/Util/InputValidator.java

```diff
@@ -0,0 +1,77 @@
+package lotto.Util;
+
+import static lotto.Util.LottoOption.*;
+import static lotto.Util.ValidatorMessage.*;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class InputValidator {
+
+    public static boolean isEmpty(String input) {
+        if (input == null || input.isEmpty()) {
+            throw new IllegalArgumentException(ERROR_MESSAGE + NULL_OR_EMPTY_ERROR);
+        }
+        return true;
+    }
+
+    public static void checkDivisibleBy1000(String input) {
+        if (Integer.parseInt(input) % LOTTO_TICKET_PRICE != 0) {
+            throw new IllegalArgumentException(ERROR_MESSAGE + CAN_NOT_DIVIDE_BY_1000_ERROR);
+        }
+    }
+
+    public static boolean isNumber(String input) {
+        if (input.contains(LOTTO_NUMBER_DELIMITER)) {
+            input = input.replaceAll("[,\\\\s]+", "");
+        }
+        for (char c : input.toCharArray()) {
+            if (!Character.isDigit(c)) {
+                throw new NumberFormatException(ERROR_MESSAGE + NOT_A_NUMBER_ERROR);
+            }
+        }
+        return true;
+    }
+
+    public static void checkSixNumber(String input) {
+        if (input.split(LOTTO_NUMBER_DELIMITER).length != 6) {
+            throw new IllegalArgumentException(ERROR_MESSAGE + NOT_A_SIX_DIGIT_ERROR);
+        }
+    }
+
+    public static boolean isValidRangeNumber(String input) {
+        if (!input.contains(LOTTO_NUMBER_DELIMITER)) {
+            if (Integer.parseInt(input) < LOTTO_MIN_NUMBER || Integer.parseInt(input) > LOTTO_MAX_NUMBER) {
+                throw new IllegalArgumentException(ERROR_MESSAGE + NOT_A_VALID_RANGE_NUMBER_ERROR);
+            }
+        }
+        List<String> numbers = List.of(input.split(LOTTO_NUMBER_DELIMITER));
+        for (String number : numbers) {
+            if (number == "") {
+                throw new IllegalArgumentException(ERROR_MESSAGE + NOT_ENTER_VALUE_BETWEEN_DELIMITER);
+            }
```

Comment:
```suggestion
            if (number.isBlank()) {
                throw new IllegalArgumentException(ERROR_MESSAGE + NOT_ENTER_VALUE_BETWEEN_DELIMITER);
            }
```
`String.isBlank` 를 활용하시면 `""` 뿐만 아니라 빈칸으로만 이루어진 문자열 `"     "` 와 같은 값도 같이 잡아줄 수 있습니다!😁